templates/js/copilot-plugin-from-scratch/.vscode/settings.json

```diff
@@ -12,9 +12,5 @@
         "obj": true,
         "bin": true
     },
-    "azureFunctions.postDeployTask": "npm install (functions)",
-    "azureFunctions.projectLanguage": "JavaScript",
-    "azureFunctions.projectRuntime": "~4",
-    "debug.internalConsoleOptions": "neverOpen",
-    "azureFunctions.preDeployTask": "npm prune (functions)"
+    "debug.internalConsoleOptions": "neverOpen"
```

Comment:
What's this setting for?

Reply:
I copied it from the template of azure functions, if we don't need it for TTK, I will remove it.